### DISCUSSION TITLE
Guide to use fabric together with mmengine?

### DISCUSSION THREAD

**Discussion by hiyyg** [open-mmlab/mmengine#1371] [Ideas]
Fabric can offer multiple training strategies and precisions with just changes of a few lines. Is there any way to apply fabric to mmengine's runner?

**Comment by zhouzaida** [comment]
mmengine also supports various distributed training frameworks or algorithms, such as FSDP, DeepSpeed, ColossalAI, so we don't have any plans to support Fabric for now. See more details at https://mmengine.readthedocs.io/en/latest/common_usage/large_model_training.html
